src/JsonSerialisation.ts

```diff
@@ -383,7 +383,7 @@ export class JsonSerialisation{
 		// var controlCmd = obj as ControlCommand;
 		let controlCmd = asOrNull(obj, ControlCommand);
 		if (controlCmd) {
-			return JsonSerialisation._controlCommandNames[controlCmd.commandType];
+			return JsonSerialisation._controlCommandNames[Math.floor(controlCmd.commandType)];
```

Comment:
Is the `Math.floor` call there to cast the value as an int?

Reply:
I… think so. Honestly, upon further inspection it's absolutely useless, since `ControlCommand` is an `enum`. That's a good catch, let me remove this.